options_test.go

```diff
@@ -141,3 +141,87 @@ func TestPassAfterNonOptionWithPositionalIntFail(t *testing.T) {
 		assertStringArray(t, ret, test.ret)
 	}
 }
+
+func TestTerminatedOptions(t *testing.T) {
+	type testOpt struct {
+		Args    [][]string `short:"a" long:"args" terminator:";"`
+		Verbose bool       `short:"v"`
+		Mode    int        `short:"m"`
+	}
+
+	tests := []struct {
+		parserOpts Options
+		args       []string
+		_Args      [][]string
+		_Verbose   bool
+		_Mode      int
+		ret        []string
+		shouldErr  bool
+	}{
+		{
+			args: []string{
+				"--args", "bin", "-xyz", "--foo=bar", "-m", "5", "-v", "foo bar", ";",
+				"-v",
+				"--args=", "--no-delim",
```

Comment:
Should the `--args=` form even be allowed?

Reply:
Disallowed it. I agree it gets a bit confusing.